compiler+runtime/test/cpp/jank/read/lex.cpp

```diff
@@ -1574,6 +1574,85 @@ namespace jank::read::lex
         }
       }
 
+      SUBCASE("Shebang Comments")
+      {
+        SUBCASE("Empty")
+        {
```

Comment:
All of your tests have a space after `#!`, which is not how shebangs are used by default. We should have an actual test for a real shebang.

Reply:
Gotcha.
Honestly, I did not know `#!` was a thing until this issue, so I was just copying the `;` comment tests.

Reply:
Well, I know about using them at the top of scripts... I didn't know they would be treated like a comment by clojure.

Reply:
Clojure supports them so that you can do things like:

```bash
#!/usr/bin/env bb
(println "hello")
```
And then
```bash
chmod +x hello.clj
./hello.clj
```

I suppose the easiest way for them to do that was to just treat `#!` as a comment.

Reply:
Yeah, I wasn't getting that when I was coding this. Thanks